Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -4,7 +4,7 @@
 #include <QFileDialog>
 #include <QToolButton>
 
-#include <Core/Utils/File/OBJFileManager.hpp>
+#include <Core/File/deprecated/OBJFileManager.hpp>

```

Comment:
I would personally prefer to keep the obj loader as a simple loader, very specialized but which can be useful I you don't want to use assimp.